src/main/java/com/sonymobile/tools/gerrit/gerritevents/GerritConnection.java

```diff
@@ -445,7 +448,9 @@ public void run() {
                     if (!channel.isConnected() || !sshConnection.isConnected()) {
                         throw new IllegalStateException("SSH connection is already lost.");
                     }
-                    sleep(SSH_RX_SLEEP_MILLIS);
+                    if (linecount > 0) {
```

Comment:
have you tried it in your test environment to see if there is any real performance impact?